src/utils/KML.js

```diff
@@ -262,12 +282,12 @@ const sanitizeFeature = (feature) => {
  * @param {String} kmlString A string representing a KML file.
  * @param {<ol.Projection|String>} featureProjection The projection used by the map.
  */
-const readFeatures = (kmlString, featureProjection) => {
+const readFeatures = (kmlString, featureProjection, mapResolution) => {
```

Comment:
In fact use of mapResolution is useless since we use the resolution from the styleFunction parameter. we can remove this parameter.

Reply:
Yes, you're right, it's useless